process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,87 +2,118 @@
+| targetFolderId | String | *Optional.* The target folder ID to save the document in Content Services. |
+| targetFolderPath | String | *Requires one.* The location path or relative path of the folder to store the signed document in Content Services. For example, a location path: `/app:company_home/app:user_homes/cm:hruser` or a relative path: `/User Homes/hruser`.  |
+| targetFileType | String | *Optional.* The type to set for the signed file, for example `fin:invoice`. |
+| targetFileMetadata | content-metadata | *Optional.* Metadata assigned to the signed document in Content Services. |
+| underscoreMetadata | boolean | *Optional.* If set to `true` the received prefixed properties names contain underscores (_) instead of colons (:), for separating the namespace prefix from the property name. |
+
+The output parameters from DOWNLOAD_DOCUMENT are:
+
+| Property | Type | Description |
+|--- | --- | --- |
+| file | file | The file stored in Content Services of the document. |
+
+No exceptions or errors are thrown by the connector, however all exceptions are caught and logged. The task execution is always successful, and errors will be returned in an error event.
+
+### Events
+
+The DocuSign connector produces events when the DocuSign envelope changes its status, the events are:
+
+* `ENVELOPE_VOIDED` -  The envelope has been voided by the sender or has expired. The void reason indicates whether the envelope was manually voided or expired.
+* `ENVELOPE_DECLINED` - The envelope has been declined by one of the recipients.
+* `ENVELOPE_COMPLETED` - The envelope has been completed by all of the recipients.
+* `ENVELOPE_DELIVERED` - This event is sent when all recipients have opened the envelope through the DocuSign signing website. This does not signify an email delivery of an envelope.
+* `ENVELOPE_SENT` - This event is sent when the email notification, with a link to the envelope, is sent to at least one recipient or when it is a recipient's turn to sign during embedded signing. The envelope remains in this state until all recipients have viewed the envelope.
+
+For more on events see [Webhook event triggers](https://developers.docusign.com/platform/webhooks/connect/event-triggers/)
+
+When a Process Automation process is instantiated this way, the following variables are populated:
+
+| Property | Type | Description |
+|--- | --- | --- |
+| envelopeId | string | Envelope ID of the document. |
+| documents | array | Documents related to the envelope and data related to them like uri, and id. |
+
+### DocuSign Environment Configuration
+
+The connector uses the DocuSign client library that relies on the DocuSign REST API and uses OAuth JWT for authentication, for more see [OAuth JWT](https://developers.docusign.com/esign-rest-api/guides/authentication/oauth2-jsonwebtoken){:target="_blank"}.
+
+The basic steps to achieve are:
 
-## Configuration
+1. Create a DocuSign account.
```

Comment:
Can we provide a direct link to help customers to create a DocuSign account?

Reply:
done